### PR TITLE
Update Overview pages (WAF, FW, Ruleset Engine)

### PR DESCRIPTION
**Pull request by pedrosousa**
Adds content to the first page (Overview page) of Firewall Rules, WAF, and Ruleset Engine.

Addresses:
https://jira.cfops.it/browse/PCX-2233
https://jira.cfops.it/browse/PCX-2257
https://jira.cfops.it/browse/PCX-2616

### CODE REVIEW

File: products/waf/src/content/index.md [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
+# Cloudflare Web Application Firewall
 
-<ContentColumn>
+The Cloudflare Web Application Firewall (WAF) provides both automatic protection from vulnerabilities and the flexibility to create custom rules.
 
-The Cloudflare Web Application Firewall (WAF) provides both automatic protection from vulnerabilities and the flexibility to create custom WAF rules.
+## Main features
+
+* **Custom Rules**: Create your own Custom Firewall Rules to protect your website and your APIs from malicious incoming traffic.
+* **Rate Limiting rules**: Define rate limits for incoming requests matching an expression and the action to take when those rate limits are reached.
```
Comment (marciocloudflare) [inline products/waf/src/content/index.md]:
```suggestion
* **Rate Limiting rules**: Define rate limits for incoming requests matching an expression, and the action to take when those rate limits are reached.
```

**Review by marciocloudflare — APPROVED** [review]

File: products/waf/src/content/index.md [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
+* **WAF Managed Rulesets**: Enable the pre-configured Managed Rulesets to get immediate protection. These rulesets are regularly updated, offering advanced zero-day vulnerability protections. Adjust the behavior of managed rules, choosing from several possible actions.
+* **Exposed Credential Checks**: Monitor and block use of stolen/exposed credentials for account takeover.
+* **Firewall Analytics**: Identify and investigate security threats using an intuitive interface. Tailor your security configurations based on the activity log.
+
+## Availability
+
+The Cloudflare WAF is available to all Cloudflare plans, but the exact features and limits depend on your current plan. Rate limiting is a paid add-on on all plans.
```
Comment (migueldemoura) [inline products/waf/src/content/index.md:21]: I guess we need to update the marketing page, as it still uses the term `WAF` for `WAF Managed Rulesets` and that's only available in paid plans.
https://www.cloudflare.com/plans/

**Review by migueldemoura — APPROVED** [review]

File: products/waf/src/content/index.md (hunk 2 above)
Comment (pedrosousa) [reply products/waf/src/content/index.md:21]: Thanks, I'll look into it. I believe the same also happens with at least one KB page that I will address soon.

Comment (pedrosousa) [reply products/waf/src/content/index.md:21]: Update: After discussing it with Michael, I'll update this sentence in a separate PR.

**Pull request merged**